wallet/wallet.go

```diff
@@ -3100,7 +3100,17 @@ func (w *Wallet) SendOutputs(outputs []*wire.TxOut, account uint32,
 		return nil, err
 	}
 
-	return w.publishTransaction(createdTx.Tx)
+	txHash, err := w.publishTransaction(createdTx.Tx)
+	if err != nil {
+		return nil, err
+	}
+
+	// Sanity check on the returned tx hash.
+	if *txHash != createdTx.Tx.TxHash() {
```

Comment:
Is this really needed?

Reply:
Not sure about it. Don't know if there is a risk here with any of the backends?

Reply:
I doubt it, but I guess it doesn't hurt to do the check.